ci-operator/step-registry/gather/gather-network-chain.yaml

```diff
@@ -0,0 +1,6 @@
+chain:
```

Comment:
@pecameron sigh. I guess we need a `ci-operator/step-registry/gather-network` directory in which you put your stuff.